src/SqlVirtualMachine/SqlVirtualMachine/Common/ArgumentCompleters/SkuCompleter.cs

```diff
@@ -0,0 +1,33 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.SqlVirtualMachine.Common.ArgumentCompleters
+{
+    /// <summary>
+    /// Class used to automatically complete the offer provided in the cmdlet
+    /// </summary>
+    public class SkuCompleter : PSArgumentCompleterAttribute
```

Comment:
I think its a convention to name Attributes with a "Attribute" suffix. You will be able to use it with the same name even if it has the Attribute Suffix. For Eg. "SkuCompleterAttribute" can be used as [SkuCompleter]